src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -413,11 +463,15 @@ inputs = {
 }
 ```
 
-Run `terragrunt plan`. Looking at the output the `env_name` variable will now have 'dev:' added. Run `terragrunt apply` to update your configurations.
+Adding a `locals` block creates a local variable. Using the replace built-in function removes the unwanted parts of the relative path. And updating the `inputs` block to use the local variable.
```

Comment:
Can we rephrase this? The language doesn't seems to convey the context properly. I suggest something like. 
```suggestion
Here, you added a `locals` block to create a local variable and used the built-in `replace` function to remove the unwanted parts of the relative path.  Then, you updated the `inputs` block to use the local variable.
```

Reply:
Agreed. I mostly copied this from above and changed the commas to periods. I should have edited the language. Good catch!